package.json

```diff
@@ -42,5 +42,11 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/snyk/policy.git"
+  },
+  "tap": {
+    "branches": 85,
+    "functions": 95,
+    "lines": 90,
+    "statements": 90
```

Comment:
Coverage is now on by default. Setting this to current levels as it would be good not to drop during refactoring https://node-tap.org/changelog/#breaking-changes